mlir/include/mlir/Dialect/SCF/Transforms/TileUsingInterface.h

```diff
@@ -97,6 +97,30 @@ struct SCFTileAndFuseOptions {
     tilingOptions = options;
     return *this;
   }
+
+  /// Control function to check if a slice needs to be fused or not,
+  /// The control function recieves
+  /// 1) the slice along which fusion is to be done,
+  /// 2) the producer value that is to be fused
+  /// 3) a boolean value set to `true` if the fusion is from
+  ///    a destination operand.
+  /// It retuns two booleans
+  /// - returns `true` if the fusion should be done through the candidate slice
+  /// - returns `true` if a replacement for the fused producer needs to be
```

Comment:
Same here. Why do you want to have this information? What is the implication of always returning false if we don't provide a callback (i.e., using fusion through transform?)?

Reply:
If it is `false` (which is the default) the producers just get fused into the tile loop nest. If you look at https://github.com/llvm/llvm-project/blob/main/mlir/test/Interfaces/TilingInterface/tile-fuse-and-yield-using-interface.mlir you will see an example where it is useful for this to be `true`.